src/Header.js

```diff
@@ -3,7 +3,9 @@ import './Header.css';
 
 function Header() {
```

Comment:
We could probably just do the header directly in App, but for now it seems good to have a separate component?

Reply:
yeah, if it's not going to hold any data or have any methods, it shouldn't be a component